src/Behat/Behat/Output/Printer/Formatter/ConsoleFormatter.php

```diff
@@ -30,7 +30,7 @@ final class ConsoleFormatter extends BaseOutputFormatter
      */
     public function format($message): string
     {
-        return preg_replace_callback(self::CUSTOM_PATTERN, array($this, 'replaceStyle'), $message);
+        return preg_replace_callback(self::CUSTOM_PATTERN, array($this, 'replaceStyle'), $message) ?? '';
```

Comment:
```suggestion
        return preg_replace_callback(self::CUSTOM_PATTERN, array($this, 'replaceStyle'), $message) ?? 'Warning: behat formatter cannot handle the output the content is too big to be printed';
```

Maybe this would be a nice addition?

Reply:
I don't think we can guarantee that it's a size issue, but I agree the message would ideally include the reason it failed rather than just an empty string.

Something including `preg_last_error_msg()` would probably do the trick, although we'd need to add symfony/polyfill-php80 until we drop support for the 7.x series